crates/dekaf/src/read.rs

```diff
@@ -119,7 +172,27 @@ impl Read {
 
             // Encode the key.
             let key = if is_control {
-                None
+                // From https://cwiki.apache.org/confluence/display/KAFKA/KIP-98+-+Exactly+Once+Delivery+and+Transactional+Messaging
```

Comment:
@jgraettinger I did more digging into how control messages are implemented, and came up with this. Materialize is able to read through the end of a slowly changing collection, and Tinybird doesn't complain about null documents. Anything using librdkafka will show debug messages like this:

```
%7|1726574941.686|MSG|rdkafka#consumer-2|: demo.2Fwikipedia.2Frecentchange.2Dsampled [0]: Skipping ctrl msg with unsupported version 9999 at offset 14536044024
```

But other than that, completely ignore them. I think this is the ideal solution.